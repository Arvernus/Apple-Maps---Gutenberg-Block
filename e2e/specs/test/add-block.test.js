/**
 * WordPress dependencies
 */
import {
	createNewPost,
	activatePlugin,
	insertBlock,
	getEditedPostContent,
	publishPost,
} from '@wordpress/e2e-test-utils';

describe( 'Add Block', () => {
	it( 'Can Add Block', async () => {
		await activatePlugin( 'arvernus-apple-maps-block' );
		await createNewPost();
		await insertBlock( 'Mapkit' );
		await publishPost();
		expect( await getEditedPostContent() ).toMatchSnapshot();
		await browser.close();
	} );
} );
