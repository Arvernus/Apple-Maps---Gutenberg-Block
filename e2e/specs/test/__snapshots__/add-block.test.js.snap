// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add Block Can Add Block 1`] = `
"<!-- wp:mapkitjs/map -->
<div class=\\"wp-block-mapkitjs-map\\" id=\\"map\\" data-shows-map-type-control=\\"true\\" data-map-type=\\"standard\\" data-point-title=\\"221B Baker Street\\" data-point-subtitle=\\"London\\" data-point-glyph-text=\\"\\" data-point-latitude=\\"51.5237503\\" data-point-longitude=\\"-0.158555\\" data-point-color=\\"#000000\\"></div>
<!-- /wp:mapkitjs/map -->"
`;
